components/Coverups/CoverupsTopDescription.jsx

```diff
@@ -0,0 +1,12 @@
+import TopPageDescription from '../TopPageDescription/TopPageDescription';
```

Comment:
```suggestion
import TopPageDescription from '~/components/common/TopPageDescription/TopPageDescription';
```
чтобы была одинаковая структура

Reply:
Поменял, проверил другие свои компоненты, пока вроде все ок